### PR TITLE
TN-351 formatted fhir.py, group.py, i18n.py and identifier.py

### PR DESCRIPTION
**Pull request by John0Randolph**

### CODE REVIEW

File: portal/models/fhir.py [hunk 1]
```diff
@@ -149,7 +153,7 @@ def __str__(self):
         """Print friendly format for logging, etc."""
         components = ','.join([str(x) for x in
```
Comment (pbugni) [inline portal/models/fhir.py]: newline after `join(` should make for better alignment

File: portal/models/fhir.py [hunk 2]
```diff
@@ -519,14 +529,12 @@ def consolidate_answer_pairs(answers):
         answer_types = [a.keys()[0] for a in answers]
 
         # Exit early if assumptions not met
-        if (
-            len(answers) % 2 or
-            answer_types.count('valueCoding') != answer_types.count('valueString')
-        ):
+        if (len(answers) % 2 or answer_types.count('valueCoding')
```
Comment (ivan-c) [inline portal/models/fhir.py]: Please don't change the other lines unless necessary; ie keep the original, but break the long line on `!=`

File: portal/models/i18n.py [hunk 3]
```diff
@@ -90,11 +92,11 @@ def upsert_to_template_file():
 
     try:
         with open(
-            os.path.join(
-                current_app.root_path,
-                "translations/messages.pot",
-            ),
-            "r+",
+                os.path.join(
```
Comment (ivan-c) [inline portal/models/i18n.py]: Too much indentation

File: portal/models/i18n.py [hunk 4]
```diff
@@ -134,16 +137,19 @@ def fix_references(pot_fpath):
     base_url = "%s/tree/develop" % current_app.config.metadata['home-page']
 
     with open(pot_fpath) as infile, tempfile.NamedTemporaryFile(
-        prefix='fix_references_',
-        suffix='.pot',
-        delete=False,
+            prefix='fix_references_',
```
Comment (ivan-c) [inline portal/models/i18n.py]: Too much indentation

File: portal/models/fhir.py [hunk 5]
```diff
@@ -540,8 +551,9 @@ def consolidate_answer_pairs(answers):
     def entry_method(row_data, qnr_data):
         # Todo: replace with EC.PAPER CodeableConcept
         if (
-            'type' in qnr_data['encounter'] and
-            'paper' in (c.get('code') for c in qnr_data['encounter']['type'])
+                'type' in qnr_data['encounter'] and
```
Comment (ivan-c) [inline portal/models/fhir.py]: Too much indentation

**Pull request merged**